portfolio/src/main/webapp/style.css

```diff
@@ -162,7 +162,8 @@ a:hover {color:#560d32;}
   display: block;
   margin: auto;
   max-width: 700px;
-  height: 75%;
+  max-height: 75%;
```

Comment:
nit pick, but looks like this out of alphabetical order :) (feel free to merge these changes and fix this in the next pull request)